addons/interactions/src/Panel.tsx

```diff
@@ -154,6 +162,10 @@ export const Panel: React.FC<AddonPanelProps> = (props) => {
       goto: (callId: string) => emit(EVENTS.GOTO, { storyId, callId }),
       next: () => emit(EVENTS.NEXT, { storyId }),
       end: () => emit(EVENTS.END, { storyId }),
+      rerun: () => {
+        setIsRerunAnimating(true);
```

Comment:
I assume from the previous situation that `if (!storyId) return` was needed because this button is rendered at `TOOL`, but now it is under `PANEL` and It seems like `storyId` exists always thus I remove this guard.